lib/node_modules/@stdlib/stats/base/ndarray/meankbn/docs/types/test.ts

```diff
@@ -0,0 +1,57 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable space-in-parens */
+
+import zeros = require( '@stdlib/ndarray/zeros' );
+import meankbn = require( './index' );
+
+
+// TESTS //
+
+// The function returns a number...
+{
+	const x = zeros( [ 10 ], {
+		'dtype': 'generic'
+	});
+
+	mean( [ x ] ); // $ExpectType number
```

Comment:
```suggestion
	meankbn( [ x ] ); // $ExpectType number
```
Applies here and for function calls below in this file